CONTRIBUTING.md

```diff
@@ -110,3 +110,103 @@ If you need to submit a patch that changes only data but does not alter the sche
+  * Wrong:
+  ```python
+  sql = ("SELECT processed_data_status FROM qiita.processed_data_status pds JOIN "
+         "qiita.processed_data pd USING (processed_data_status_id) JOIN "
+         "qiita.study_processed_data spd USING (processed_data_id) "
+         "WHERE spd.study_id = %s")
+  ```
+  * Correct:
+  ```python
+  sql = """SELECT processed_data_status
+           FROM qiita.processed_data_status pds
+              JOIN qiita.processed_data pd USING (processed_data_status_id)
+              JOIN qiita.study_processed_data spd USING (processed_data_id)
+           WHERE spd.study_id = %s"""
+  sql = "SELECT * FROM qiita.qiita_user"
+  ```
+3. Use PEP8-style indentation for the SQL queries, as it will improve readability. Common SQL best-practices recommend to have a new line for the `SELECT`, `FROM`, `WHERE` and similar clauses:
+  * Wrong:
+  ```python
+  sql = """SELECT udt_name FROM information_schema.columns WHERE 
+           column_name = %s AND table_schema = 'qiita' AND (table_name = %s
+           OR table_name = %s)"""
+  ```
+  * Correct:
+  ```python
+  sql = """SELECT udt_name
+           FROM information_schema.columns
+           WHERE column_name = %s AND table_schema = 'qiita'
+               AND (table_name = %s OR table_name = %s)"""
+  ```
+4. Never, NEVER, use python string formatting to complete the SQL query parameters. Use the `sql_args` parameter from the transaction object. This is a strong recommendation from the psycopg2 developers to avoid SQL injection attacks ([see](http://initd.org/psycopg/docs/usage.html#the-problem-with-the-query-parameters) for more information):
+  * Wrong:
+  ```python
+  sql = """SELECT processed_data_status
+           FROM qiita.processed_data_status pds
+              JOIN qiita.processed_data pd USING (processed_data_status_id)
+              JOIN qiita.study_processed_data spd USING (processed_data_id)
+           WHERE spd.study_id = %s""" % study.id
+  with TRN:
+      TRN.add(sql)
+  ```
+  * Correct:
+  ```python
+  sql = """SELECT processed_data_status
+           FROM qiita.processed_data_status pds
+              JOIN qiita.processed_data pd USING (processed_data_status_id)
+              JOIN qiita.study_processed_data spd USING (processed_data_id)
+           WHERE spd.study_id = %s"""
+  with TRN:
+      TRN.add(sql, [study.id])
+  ```
+5. However, python string formatting is allowed to provide table or column names, although this should be done through the `str.format` function. Table or column names as parameters are not supported by psycopg2. Using `str.format` is desirable because if you need to pass parameters to the SQL statement, the python string formatting will fail (see second example below):
+  * Wrong:
+  ```python
+  table = "qiita_user"
+  sql = "SELECT * FROM qiita.%s" % table
+
+  # This will fail during execution
+  table = "qiita_user"
+  sql = "SELECT * FROM qiita.%s WHERE email = %s" % table
+  TypeError: not enough arguments for format string
```

Comment:
I commented it and added it below the "this will fail..." line, as I want to show the specific error that you'll receive.